ui/scripts/storage.js

```diff
@@ -68,7 +68,7 @@
                     actions: {
                         // Add volume
                         add: {
-                            label: 'label.add.volume',
+                            label: 'Add',
```

Comment:
can you revert to using label.* instead of hardcoding label values since it would break i18n?